storage/api-docs/api-docs.json

```diff
@@ -6,7 +6,7 @@
     },
```

Comment:
Is this file updated manually or automatically? If it is updated automatically, would it make sense to add it to the .gitignore?

Reply:
Think it's the auto updated one. You are right, it should be ignored.